app/socket-apis/subscribe-deliberation.js

```diff
@@ -34,30 +34,14 @@ async function subscribeDeliberation(deliberationId, requestHandler) {
           )[0]
 
           await Dturns.upsert(this.synuser.id, deliberationId, 0, round, shownStatementIds, groupings || {})
-
-          // Then broadcast the update if changes were made
-          const participants = Object.keys(Discussions[deliberationId].Uitems).length
-          const lastRound = Object.keys(Discussions[deliberationId].ShownStatements).length - 1
-
-          if (
-            lastRound != Discussions[deliberationId].lastRound ||
-            participants != Discussions[deliberationId].participants
-          ) {
-            const eventName = subscribeEventName('subscribe-deliberation', deliberationId)
-            const updateData = {
-              participants: Object.keys(Discussions[deliberationId].Uitems).length,
-              lastRound: lastRound,
-            }
-
-            server.to(deliberationId).emit(eventName, updateData)
-
-            Discussions[deliberationId]['lastRound'] = lastRound
-            Discussions[deliberationId]['participants'] = participants
-          }
         },
         getAllUInfo: async () => {
           return await Dturns.getAllFromDiscussion()
         },
+        updates: async updateData => {
+          const eventName = subscribeEventName('subscribe-deliberation', deliberationId)
```

Comment:
This works, but if the const eventName line is up, outside of the function call, then it won't have to be calculated every time the function is called.  And it this case it's not something that changes. It's just a performance optimization, but wanted to share it with you.